tests/test_adapter_plugin.py

```diff
@@ -190,13 +190,33 @@ def test_find_adapter_by_name(self):
         )
 
     def test_find_manifest_by_environment_variable(self):
-        # write the file into a temp dir
         suffix = ".plugin_manifest.json"
+
+        # back up existing manifest
+        bak = otio.plugins.manifest._MANIFEST
+        bak_env = os.environ.get('OTIO_PLUGIN_MANIFEST_PATH')
+
+        # Generate a fake manifest in a temp file, and point at it with
+        # the environment variable
         with tempfile.NamedTemporaryFile(suffix=suffix) as fpath:
-            os.environ['OTIO_PLUGIN_MANIFEST_PATH'] = fpath.name + ':foo'
             otio.adapters.write_to_file(self.man, fpath.name, 'otio_json')
+
+            # clear out existing manifest
+            otio.plugins.manifest._MANIFEST = None
+
+            # set where to find the new manifest
+            os.environ['OTIO_PLUGIN_MANIFEST_PATH'] = fpath.name + ':foo'
             result = otio.plugins.manifest.load_manifest()
-            self.assertEqual(result.media_linkers[0].name, "example")
+
+            # Rather than try and remove any other setuptools based plugins
+            # that might be installed, this check is made more permissive to
+            # see if the known unit test linker is being loaded by the manifest
+            self.assertTrue(len(result.media_linkers) > 0)
+            self.assertIn("example", (ml.name for ml in result.media_linkers))
+
+        otio.plugins.manifest._MANIFEST = bak
+        if bak_env:
+            os.environ['OTIO_PLUGIN_MANIFEST_PATH'] = bak_env
```

Comment:
Don't you need to add an else clause to restore original state if this envvar was not originally set?
```suggestion
            os.environ['OTIO_PLUGIN_MANIFEST_PATH'] = bak_env
```
```suggestion
        else:
            os.environ['OTIO_PLUGIN_MANIFEST_PATH'] = bak_env
```